examples/tts/conf/fastpitch/fastpitch_22050.yaml

```diff
@@ -0,0 +1,286 @@
+# This config contains the default values for training an English 22.05kHz FastPitch model.
+# If you want to train a model on other dataset, you can change config values according to your dataset.
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+max_epochs: ???
+batch_size: 32
+weighted_sampling_steps_per_epoch: null
+
+n_speakers: ???
+speaker_path: null
+feature_stats_path: null
+
+train_ds_meta: ???
+val_ds_meta: ???
+log_ds_meta: ???
+
+phoneme_dict_path: ???
+heteronyms_path: ???
+
+log_dir: ???
+vocoder_type: ???
+vocoder_name: null
+vocoder_checkpoint_path: null
+
+# The below feature config should match the feature.yaml config used during preprocessing.
+sample_rate: 22050
+win_length: 1024
+hop_length: 256
+
+mel_feature:
+  _target_: nemo.collections.tts.parts.preprocessing.features.MelSpectrogramFeaturizer
+  sample_rate: ${sample_rate}
+  win_length: ${win_length}
+  hop_length: ${hop_length}
+  mel_dim: 80
+  lowfreq: 0
+  highfreq: 8000
```

Comment:
It was discussed before, and I remembered we decided to go with `null` for 44100kHz config, while keep `8000` unchanged for 22050kHz. Is it a good time to switch to `null` for 22050kHz config?